src/diffusers/schedulers/scheduling_ddpm.py

```diff
@@ -405,6 +406,71 @@ def step(
 
         return DDPMSchedulerOutput(prev_sample=pred_prev_sample, pred_original_sample=pred_original_sample)
 
+    def batch_step_no_noise(
+        self,
+        model_output: torch.FloatTensor,
+        timesteps: List[int],
+        sample: torch.FloatTensor,
+    ) -> Union[DDPMSchedulerOutput, Tuple]:
+        """
+        Predict the previous timestep of multiple samples in batch. Does not add noise.
```

Comment:
Same as https://github.com/huggingface/diffusers/pull/3716/files#r1227448253. But also, I think we're missing docstrings.